src/program.js

```diff
@@ -621,6 +627,14 @@ Example: $0 --help run.
         type: 'string',
         requiresArg: true,
       },
+      'fennec-mode': {
+        describe: (
+          'Control the APK as a Fennec-based browser (defaults to true, ' +
+          'set to false if the APK is a geckoview-based browser)'
```

Comment:
nit: `GeckoView`.

Why is the default encoded in a comment instead of `default: true`?

Reply:
> nit: GeckoView.

:+1:

> Why is the default encoded in a comment instead of default: true?

Because the default `fennecMode` value is actually meant to be `undefined` for now, and in that case we still defaults to the "Fennec compatibility mode" unless the APK name contains ".fenix" in its name.

> Fenix is a moving target, whereas we know what Fennec looks like. In the long term, we should be able to assume Fenix by default, unless.
> ...

Once we are going to be ready to "assume Fenix by default", we can change change the `fennecMode` option definition to have `false` as its default value, and the user may only need to use it to force the legacy "Fennec compatibility mode" if ever necessary.

I'm going to file an issue, to make this change when we are ready to default to `fennecMode: false`.